lib/row.rb

```diff
@@ -0,0 +1,125 @@
+require 'field_serializer'
+require 'nokogiri'
+
+class String
+  def tidy
+    gsub(/[[:space:]]+/, ' ').strip
+  end
+end
+
+class Row
+  include FieldSerializer
+
+  def initialize(tds)
+    @tds = tds
+  end
+
+  field :id do
+    tds[0].xpath('a/@title')
+          .text
+          .downcase
+          .gsub(/ /,'_')
+          .gsub('_(page_does_not_exist)','')
+  end
```

Comment:
Is every person guaranteed to have a link? It's fairly common on Wikipedia for a person to simply be plain text, and not linked.